guillotina/contrib/dbusers/services/groups.py

```diff
@@ -0,0 +1,81 @@
+
+from guillotina import configure
+from guillotina.api.content import DefaultDELETE
+from guillotina.api.content import DefaultPATCH
+from guillotina.api.service import Service
+from guillotina.component import get_multi_adapter
+from guillotina.interfaces import IPATCH
+from guillotina.interfaces import IContainer
+from guillotina.interfaces import IResourceSerializeToJsonSummary
+from guillotina.response import HTTPNotFound
+from guillotina.utils import navigate_to
+from zope.interface import alsoProvides
+
+
+@configure.service(
+    for_=IContainer, method="GET",
+    name="@groups"
+)
+class GetGroups(Service):
+    async def __call__(self):
+        groups = await self.get_groups_from_folder()
+
+        result = []
+        for group in groups:
+            serializer = get_multi_adapter(
+                (group, self.request), IResourceSerializeToJsonSummary
+            )
+            result.append(await serializer())
+
+        return result
+
+    async def get_groups_from_folder(self):
+        group_folder = await navigate_to(self.context, "groups")
+        groups = []
+        async for _, group in group_folder.async_items():
```

Comment:
Should we do a search?

Reply:
It started with a search, but later, @vangheem removed it https://github.com/plone/guillotina_dbusers/commit/e3d84fe1c72a6df796aa2ecb0a550a9277e9a5e8

Reply:
Yes, sorry. get_by_type was removed. Do you think we need it?

Reply:
Can simply be done with .search(container, {type_name:})

Sorry, traveling so syntax is not perfect

Reply:
Is the catalog installed by default?

Reply:
No, you will need to check if it is configured before using

Reply:
Agree with .search

Reply:
@ferran can you try to load search utility and if its not then do this fallback ?

Reply:
@bloodbare @vangheem any examples on a catalog search by type_name that works both elastic and pgcatalog?

Reply:
Yup, the new query catalog works in a generic way to parse queries!

just do...
```
await catalog.query(container, {"portal_type": "User"})
```

Reply:
oh cool! thanks

Reply:
I am now able to get users from the search.

Should I then get them individually from storage and output their summaryJSONSerialization?

Reply:
fixed!

Reply:
ready for re-review